li/client-php/lib/NFe/APIRequest.php

```diff
@@ -154,6 +154,12 @@ private function requestWithCURL( $method, $url, $headers, $data = array(), $pdf
     }
 
     curl_close($curl);
+
+    $logger = wc_get_logger();
+		$logger->info('$response_body ='.$response_body, array( 'source' => 'price-changes' ) );
```

Comment:
@souzadavi @JoaoVictorLNacional podemos usar aqui o $logger->debug, acredito que possa ser interessante para não afetar a performance nos cenários onde não existe nenhum problema.

talvez podemos criar uma lógica, onde temos que usar o $logger->info sempre que o response for diferente de sucesso.